src/Gestures/Gesture.vala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 elementary, Inc (https://elementary.io)
+ *           2020 José Expósito <jose.exposito89@gmail.com>
```

Comment:
```suggestion
 * Copyright 2021 elementary, Inc (https://elementary.io)
 *           2021 José Expósito <jose.exposito89@gmail.com>
```

Happy new year 😜

Reply:
Happy new year! 🤣 Still work in progress, I wish this was the only issue with this PR at the moment 🐛